Zend/zend_compile.c

```diff
@@ -9351,6 +9351,11 @@ static void zend_compile_class_decl(znode *result, zend_ast *ast, bool toplevel)
 		ce->ce_flags |= ZEND_ACC_TOP_LEVEL;
 	}
 
+	if (UNEXPECTED(zend_hash_exists(&ce->function_table, ZSTR_KNOWN(ZEND_STR_SLEEP)) && ce->__serialize == NULL)) {
```

Comment:
Don't think we need `UNEXPECTED` in the compiler (it is not used at runtime). Swapping around the conditions might also make this cheaper:


```suggestion
	if (ce->__serialize == NULL && zend_hash_exists(&ce->function_table, ZSTR_KNOWN(ZEND_STR_SLEEP))) {
```

Reply:
Indeed, just pushed a change for that